src/lib/parser/core/GameConfigRegistry.ts

```diff
@@ -51,6 +51,33 @@ export class GameConfigRegistry {
     return [...this.configs]
   }
 
+  /**
+   * Get a specific config by name
+   */
+  static getConfig (name: string): GameConfig | null {
+    const registry = gameConfigRegistry
+
+    // Try to find and instantiate config by name
+    for (const ConfigClass of registry.configs) {
+      try {
+        const config = new ConfigClass()
+        if (name === 'emerald' && config.name.toLowerCase().includes('emerald')) {
+          return config
+        }
+        if (name === 'vanilla' && config.name.toLowerCase().includes('vanilla')) {
+          return config
+        }
+        if (name === 'quetzal' && config.name.toLowerCase().includes('quetzal')) {
+          return config
+        }
+      } catch {
+        continue
+      }
+    }
+
+    return null
+  }
```

Comment:
lets think about if there is a cleaner way to do this. check where this function is used and come up with a better way that integrated better with the config system we already have